app/_hub/kong-inc/ai-proxy/how-to/_sdk-usage.md

```diff
@@ -0,0 +1,253 @@
+---
+title: SDK Usage
+nav_title: Using Programmatic SDKs with AI Proxy
+---
+
+You can use an OpenAI-compatible SDK with the AI Proxy plugin in multiple ways, depending on the required use case.
+
+| You want to... | Then use... |
+| -------------- | ----------- |
+| Proxy an unsupported model, like Whisper-2. | [OpenAI-compatible SDK for unsupported models](#using-an-unsupported-model) |
+| etc | etc |
```

Comment:
@tysoekong I added a use case table here (similar to the one we have here: ) because I thought it might be helpful to users to https://docs.konghq.com/konnect/analytics/#analyze-api-usage-and-performance-data. I wasn't sure what the use cases were here, other than the unsupported model one makes sense to me. For example, what's a use case where someone would want to use multiple models on the same provider?

Reply:
I've done this now